### DISCUSSION TITLE
Dont show all users as default

### DISCUSSION THREAD

**Discussion by Y0ngg4n** [dani-garcia/vaultwarden#5028] [Ideas]
When having many users the user page can take very long to load as the default is to show all users. The default should be around 100 or so.

**Comment by BlackDex** [comment]
This project does not maintain or develop the clients (including the web-vault).
But i think there was a newer web-vault which added this feature already. But unfortunately we can't use newer web-vault versions yet because we need some major changes to be done on the server side for those to work.

**Comment by Y0ngg4n** [comment]
@BlackDex thank you for that information. Is there a rough ETA for the changes on the backend?

**Reply by BlackDex** [reply]
No. I have not started looking at it at all right now. So no clue.
I have some busy days the last few and coming weeks.

**Reply by Y0ngg4n** [reply]
yeah alright don´t wanted to put pressure on you.
